package.json

```diff
@@ -2,33 +2,25 @@
   "name": "w3up",
   "version": "0.0.0",
   "private": true,
-  "workspaces": [
-    "packages/*"
-  ],
   "packageManager": "pnpm@9.8.0+sha256.56a9e76b51796ca7f73b85e44cf83712862091f4d498c0ce4d5b7ecdc6ba18f7",
   "scripts": {
-    "format": "prettier --write '**/*.{js,ts,yml,json}' --ignore-path .gitignore",
-    "format:check": "prettier --check '**/*.{js,ts,yml,json}' --ignore-path .gitignore",
-    "clean": "rm -rf docs node_modules pnpm-lock.yaml packages/*/{pnpm-lock.yaml,coverage,.nyc_output,dist,node_modules,.next}",
-    "lint": "pnpm -r --if-present run lint",
-    "test": "pnpm -r --if-present run test",
-    "check": "pnpm -r --if-present run check",
-    "build": "pnpm -r --if-present run build",
-    "dev": "pnpm -r --if-present --parallel run dev --preserveWatchOutput",
+    "lint": "eslint *.{js,ts} scripts && prettier --check '**/*.{js,ts,yml,json}' --ignore-path .gitignore",
+    "lint:fix": "eslint --fix *.{js,ts} scripts && prettier --write '**/*.{js,ts,yml,json}' --ignore-path .gitignore",
     "docs": "pnpm run build && typedoc --plugin typedoc-plugin-missing-exports --out docs",
     "docs:markdown": "pnpm run build && docusaurus generate-typedoc"
   },
   "devDependencies": {
-    "@arethetypeswrong/cli": "^0.15.3",
     "@docusaurus/core": "^3.0.0",
     "@docusaurus/preset-classic": "^3.0.0",
+    "@nx/js": "^20.3.2",
```

Comment:
just curious what's the difference between these two NX packages?

Reply:
[`nx`](https://www.npmjs.com/package/nx) is Nx itself; [`@nx/js`](https://www.npmjs.com/package/@nx/js) is the plugin for JavaScript/TypeScript projects. I believe it's `@nx/js` that infers Nx tasks from npm scripts, for instance.